cmake/FindUnwind.cmake

```diff
@@ -0,0 +1,56 @@
+# FindUnwind.cmake - Find libunwind package.
+#
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+#-----------------------------------------------------------------------
+# Use pkg-config to search for the module.
+#-----------------------------------------------------------------------
+find_package(PkgConfig)
+
+pkg_check_modules(UNWIND QUIET libunwind)
+
+#-----------------------------------------------------------------------
+# Find libraries and include directories.
+#-----------------------------------------------------------------------
+find_path(UNWIND_INCLUDE_DIR
+    NAMES "unwind.h"
+    PATHS ${UNWIND_INCLUDE_DIRS}
+)
+
+find_library(UNWIND_LIBRARY
+    NAMES unwind
+    PATHS ${UNWIND_LIBRARY_DIRS}
+)
+
+#-----------------------------------------------------------------------
+# Get version number
+#-----------------------------------------------------------------------
+if(UNWIND_VERSION)
+    set(UNWIND_VERSION ${UNWIND_VERSION} CACHE STRING "libunwind version")
+endif()
+
+#-----------------------------------------------------------------------
+# Handle REQUIRED and QUIETLY arguments
```

Comment:
The syntax and semantics are described in the documentation for the `find_package` command.

This particular comment was lifted verbatim from one of Find modules I used as a model. It may or may not be an error. The argument is `QUIET`; the package file interface variable it sets is `<PackageName>_FIND_QUIETLY`. Now that I think about it, the former is probably more correct.